src/main/java/com/rackspacecloud/metrics/queryservice/controllers/QueryController.java

```diff
@@ -62,10 +57,10 @@ public QueryResult query(
      * @param tenantId TenantID used for measurement lookup in tenant database
      * @return a list of measurements for tenant
      */
-    @GetMapping("/intelligence-format-query/measurements")
+    @GetMapping("/v1.0/tenant/{tenant}/intelligence-format-query/measurements")
     @Timed(value = "query.service", extraTags = {"query.type","query.intelligence.measurements"})
     public List<?> intelligenceFormattedQueryGetMeasurements(
-            final @AuthenticationPrincipal String tenantId) { // Use repose tenantId
+            final @AuthenticationPrincipal String tenantId, final @PathVariable String tenant) { // Use repose tenantId
```

Comment:
It seems like you should just keep one or the other as a parameter. In Salus we went with the path variable since impersonation might be swapping out the "authenticated tenant"; however, there's logic to either choice.